### ISSUE TITLE
[Bug]: Manage Users - Broken Authorize User Image

### ISSUE THREAD

**Issue by erw13n** [dnnsoftware/Dnn.Platform#5805]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

After upgrade DNN from 9.11 to 9.12, all `Authorize` User image is broken, but this is not happen for all `Superusers`
![image](https://github.com/dnnsoftware/Dnn.Platform/assets/792847/fc64bfae-dd38-4bca-8bbc-1675dd64f439)

As displayed on the screenshot, the one that had issue was Authorized user, but all of this users don't have a picture so by default it should display the default picture.

All other picture displayed was Superusers account.

We've upgrade 20 sites, and all had same behavior.

### Case 1:
1. Upgrade site to 9.12.0
2. All authorized users picture is broken, except Superusers

---
### Case 2:
1. Install new DNN 9.12.0
2. On create new user, the user image is broken, `but image is displayed after refresh the Website`

![image](https://github.com/dnnsoftware/Dnn.Platform/assets/792847/0bff4349-f7aa-4778-9074-c80021e6360e)

### Current Behavior

Authorize user image is displaying incorrect.

All broken image url `/DnnImageHandler.ashx?mode=profilepic&userId=[userId]&h=64&w=64` is return 403

### Affected Versions

9.12.0 (latest release)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mitchelsellers** [comment]
@valadas I thought we had a fix for this, but I cannot seem to find it.

**Comment by bdukes** [comment]
These images were introduced in https://github.com/dnnsoftware/Dnn.Platform/pull/5692, looks like some of the URLs are invalid

**Comment by valadas** [comment]
Yep #5630 came for an Evoq backend fix of a feature not present on platform. #5692 exposed that in the Users UI and I did some quick testing but honestly I never tried with different type of users and upgrade scenarios. Sooo there may be some bug here in addition to those 2 PRs.

**Comment by enfJoao** [comment]
This is an issue for me in 9.13.02
No images and the link:
https://su.investigar.pt/DnnImageHandler.ashx?mode=profilepic&userId=1021&h=64&w=64
Returns 403
